src/common/pages/database-page.tsx

```
@@ -166,39 +166,34 @@ export function DatabasePage() {
-                <TableRow key={character.id}>
-                  <TableCell>
-                    {(charactersPage - 1) * itemsPerPage + index + 1}
-                  </TableCell>
-                  <TableCell>{`${character.first_name} ${character.last_name}${
-                    character.id == game.gameState.player_id ? " (you)" : ""
-                  }`}</TableCell>
-                  <TableCell>{character.previous_job}</TableCell>
-                  <TableCell>{character.backstory}</TableCell>
-                  <TableCell>
+        <div className="grid grid-cols-2 gap-4">
+          {charactersData.map((character, index) => {
```

ESLint

[error] 'index' is defined but never used.

(@typescript-eslint/no-unused-vars)